CHANGELOG.md

```diff
@@ -1,13 +1,13 @@
 # Confluent's Golang client for Apache Kafka
 
-## v1.9.3
+# v2.0.0
 
 This is a maintenance release:
 
  * Added SetSaslCredentials. This new method (on the Producer, Consumer, and
    AdminClient) allows modifying the stored SASL PLAIN/SCRAM credentials that
    will be used for subsequent (new) connections to a broker.
-
+ * Channel based producer (Producer `ProduceChannel()`) and channel based consumer (Consumer `Events()`) are deprecated.
```

Comment:
Try to keep below 80 columns in width to make it palatable on terminals and GH code reviews.
Keep two newlines to separate from the next release.

Reply:
Noted, I'll keep this in mind for the future. Generally, I try to stick to what the file already seems to follow.